2025/08/27 20:58:18 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.5:9000", host: "127.0.0.1:8088", referrer: "https://studio.firebase.google.com/"
2025/08/27 20:58:18 [error] 29#29: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.6:9000", host: "127.0.0.1:8080", referrer: "https://studio.firebase.google.com/"
